bfvmm/src/memory_manager/src/Makefile

```diff
@@ -60,6 +60,7 @@ CROSS_OUTDIR:=../../../bin
 ################################################################################
 
 SOURCES+=memory_manager.cpp
+HEADERS=
```

Comment:
Cruft from me excising the paging code, and not trusting git after it destroyed my rebase yesterday afternoon.